packages/common/components/content/blocks/page-body.marko

```diff
@@ -33,6 +33,6 @@ $ const displayMedia = displayPrimaryImage || content.embedCode;
   </if>
   <cms-content-body tag="div" block=block obj=content />
   <if(content.type === "whitepaper")>
-    <endeavor-content-block-download-button content=content />
+    <endeavor-content-block-gated-download content=content />
```

Comment:
This should be a more generic name... it isn't _always_ gated. Instead it should be something like `whitepaper-download`